### PR TITLE
footer section completed

### LINKED ISSUE #72: Design a Footer
**Description**

Design the Footer for PDA website

**Objective**

Footer should contain 
- PDA logo (or PDA name and small description) 
- social media icons
- Quick links
- location , phone number and mail id
- Sign Up button
- Copyrights 

Files to be edited
- footer.ejs (in views folder)
- and public files (js/css/img  if Neccessary)

### PR DESCRIPTION
**Pull request by R-V-Vighnesh**
**Description**

This PR fixes #72 

**Notes for Reviewer**

the footer contains :

- PDA logo (PDA name and small description)
- social media icons
- Quick links
- location , phone number and mail id
- Sign Up button
- Copyrights

1. footer contents have been grouped into 4 sections with appropriate headings
2. scroll to top button has been designed and placed in the footer
3. footer has been made responsive
4. simple animations has have been implemented when buttons, links are hovered upon

- [ ] Frontend (Signup/Login)
- [ ] Backend (Login)
- [ ] Backend (Signup)
- [x] Homepage
- [ ] Others

teamwork by myself, @bala418 , @Bhuvanaa28
PDA_footer_updated

### CODE REVIEW

File: views/partial/footer.ejs [hunk 1]
```diff
@@ -1,2 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>PDA MIT</title>
+    <!-- Pre-build CSS -->
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="style.css">
```
Comment (coding-geek21) [inline views/partial/footer.ejs:10]: Change this to 
`<link rel="stylesheet" href="css/footer.css">`

File: views/partial/footer.ejs [hunk 2]
```diff
@@ -1,2 +1,105 @@
...
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+	<link href="http://fonts.googleapis.com/css?family=Cookie" rel="stylesheet" type="text/css">
+</head>
+<body>
+
+    <!--The text and break tags below are placed just for testing purpose. They can be removed later-->
+    <h1>Scroll down to view the footer</h1><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
```
Comment (coding-geek21) [inline views/partial/footer.ejs:17]: Also remove this line

File: views/partial/footer.ejs [hunk 3]
```diff
@@ -1,2 +1,105 @@
...
+
+    <footer class="footer-distributed">
+
+        <div class="footer-left">
+    
+            <img src="public\img\pda-logo.png" alt="logo" class="logo">
```
Comment (coding-geek21) [inline views/partial/footer.ejs:23]: `logo`

Also fix the the logo image .. It looks shrinken 

image

File: views/partial/footer.ejs [hunk 4]
```diff
@@ -1,2 +1,105 @@
...
+                <li>
+                    <a href="#">Contact</a>
+                </li>
+            </ul>
+            
+        </div>
+    
+        <div class="footer-center-2">
+            
+            <h3>Get in Touch</h3>
```
Comment (coding-geek21) [inline views/partial/footer.ejs:64]: Place some gap below the title **Get in Touch**

image

File: views/partial/footer.ejs [hunk 5]
```diff
@@ -1,2 +1,105 @@
...
+                <a href="https://www.instagram.com/pda_mit/"><i class="fa fa-instagram"></i></a>
+                <a href="https://www.linkedin.com/company/personality-development-association-mit/"><i class="fa fa-linkedin"></i></a>
+    
+            </div>
+
+            <a href="#" class="footer-sign-up">Sign up now!</a>
+            
+        </div>
+        <div class="top_scroll">
+            <span onclick="(function(){window.scrollTo({top: 0,left: 0,behavior: 'smooth'})})();"><i class="fa fa-arrow-circle-up"></i></span>
```
Comment (coding-geek21) [inline views/partial/footer.ejs:99]: Place this at the end 

image

**Review by coding-geek21 — CHANGES_REQUESTED** [review]
@R-V-Vighnesh Just minor changes to be done... Rest all looks good 👍🏼

File: views/partial/footer.ejs (hunk 1 above)
Comment (coding-geek21) [inline views/partial/footer.ejs:10]: Change this to 
`<link rel="stylesheet" href="css/footer.css">`

Comment (coding-geek21) [inline views/partial/footer.ejs:10]: Change this to
<link rel="stylesheet" href="css/footer.css">

**Comment by R-V-Vighnesh** [comment]
Snr i have made the required changes in code. Can you check whether this footer is appropriate?

PDA_footer_NewUpdates

**Comment by coding-geek21** [comment]
> Snr i have made the required changes in code. Can you check whether this footer is appropriate?
> 
> PDA_footer_NewUpdates

Looks good jnr 🤩🤩 Great work merging it ✨

**Review by coding-geek21 — APPROVED** [review]

**Pull request merged**